README.md

```diff
@@ -58,42 +67,259 @@ and modify the `system-reserved` line under `kubeletExtraArgs`:
 
+We now have Konflux up and running. Next, we shall configure Konflux to respond
+to Pull Request webhooks, build a user application and push it to a registry.
+
+### Enable Pipelines Triggering via Webhooks
+
+Pipelines Can be triggered by Pull Request activities, and their outcomes will be
+reported back to the PR page in GitHub.
+
+A GitHub app is required for creating webhooks that Tekton will listen on. When deployed
+in a local environment like Kind, GitHub will not be able to reach a service within the
+cluster. For that reason, we need to use a proxy that will listen on such events
+from within the cluster and will relay those events internally.
+
+To do that, we rely on [smee](https://smee.io/): We configure a GitHub app to send
+events to a channel we create on a public `smee` server, and we deploy a client
+within the cluster to listen to those events. The client will relay those events to
+pipelines-as-code (Tekton) inside the cluster.
+
+1. Start a new channel in [smee](https://smee.io/), and take a note of the webhook
+   proxy URL.
+
+2. Create a GitHub app following
+   [Pipelines-as-Code documentation](https://pipelinesascode.com/docs/install/github_apps/#manual-setup).
+
+   For `Homepage URL` you can insert `https://localhost:9443/` (it doesn't matter).
+
+   For `Webhook URL` insert the smee client's webhook proxy URL from previous step.
+
+   Generate and download the private key and create a secret on the cluster per PaC
+   instructions, providing the location of the private key, the App ID, and the
+   openssl-generated secret created during the process.
+
+3. Deploy the smee-client on the cluster:
+
+   Edit the [smee-client manifest](./smee/smee-client.yaml), replacing `<smee-channel>`
+   with the webhook proxy URL generated when creating the channel.
+
+   Deploy the manifest:
+
+```bash
+kubectl create -f ./smee/smee-client.yaml
+```
+
+### Onboard a new Application
+
+We now create a new GitHub repository for our application and set it up in Konflux:
+
+1. Fork the
+   [example repository](https://github.com/konflux-ci/testrepo), by
+   clicking the Fork button from that repository and following the instructions on the
+   "Create a new fork" page.
+
+   We will use our Konflux deployment to build and release Pull Requests for this fork.
+
+2. Use a text editor to edit your local copy of the
+   [example application manifests](./test/resources/demo-users/user/ns2/application-and-component.yaml):
+
+   Under the `Component` and `Repository` resources, change the `url` fields so they
+   point to your newly-created fork.
+
+   Note the format differences between the two fields! The `Component` URL has a `.git`
+   suffix, while the `Repository` URL doesn't.
+
+   Deploy the manifests:
+
+```bash
+kubectl create -f ./test/resources/demo-users/user/ns2/application-and-component.yaml
+```
+
+**Note:** Further explanation about those resources can be found in
+[later section](#create-an-application-and-component).
```

Comment:
This link opens the README again

Reply:
It should point to a section in the README. Let me have a look.

Reply:
I think I renamed the section and broke the link. Hang on.

Reply:
Should be ok now

Reply:
Yes, it is.
Thanks :)